recipe/recipe.yaml

```
@@ -0,0 +1,60 @@
+schema_version: 1
+
+context:
+  name: hepdata-validator
+  version: 0.3.6
+  python_min: 3.6
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/hepdata_validator-${{ version }}.tar.gz
+  sha256: e57b97f83509c81d579ccbbd4cdd515043ba32aa1d8543bf31a36a19dcf08fbb
+
+build:
+  number: 1
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv
+  python:
+    entry_points:
+      - hepdata-validate=hepdata_validator.cli:validate
+
+requirements:
+  host:
+    - python 3.6.*
```

```suggestion
    - python ${{ python_min }}.*
```

Fix for bug https://github.com/conda-incubator/conda-recipe-manager/issues/307